packages/uiweb/src/lib/components/space/SpaceWidget/LiveWidgetContent.tsx

```diff
@@ -444,5 +510,47 @@ const RequestsCount = styled.div`
   background-color: ${(props) => props.theme.btnColorPrimary};
   padding: 2px 4px;
   border-radius: 4px;
-  font-size: 12px;
-`;
\ No newline at end of file
+  font-size: 12px;<EmojiContainer />
```

Comment:
remove the EmojiContainer tag

Setup eslint on your VS Code to avoid pushing code like this